### PR TITLE
Adds sec magboots

### PR DESCRIPTION
**Pull request by Ktlwjec**
our lovely prebase security magboots

:cl:  
rscadd: Combat magboots can be found in the security suit storages.
/:cl:

### CODE REVIEW

File: code/game/machinery/suit_storage_unit.dm [hunk 1]
```diff
@@ -52,11 +52,12 @@
 /obj/machinery/suit_storage_unit/security
 	suit_type = /obj/item/clothing/suit/space/hardsuit/security
 	mask_type = /obj/item/clothing/mask/gas/sechailer
+	storage_type = /obj/item/clothing/shoes/magboots/security
 
 /obj/machinery/suit_storage_unit/hos
 	suit_type = /obj/item/clothing/suit/space/hardsuit/security/hos
 	mask_type = /obj/item/clothing/mask/gas/sechailer
-	storage_type = /obj/item/tank/internals/oxygen
+	storage_type = /obj/item/clothing/shoes/magboots/security
```
Comment (ghost) [inline code/game/machinery/suit_storage_unit.dm:60]: Where's the glorious //yogs?

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by Kmc2000** [comment]
This PR is the greatest thing I have ever been fortunate enough to bear witness to. The simple idea of security magboots is sublime, but simultaneously removing the oxygen tank from the security suit storage units presents truly profound implications for the meta, and balance of the game as a whole. Innovation like this surpasses the mere substrate of ss13's old, tired code, what's more --- /tg/ code. No my friends, this surpasses everything before seen and sublimates /tg/'s codebase into a well oiled machine with each beautifully individual element balanced out with precision the likes of which could have never even been conceived of by mortal minds, and yet somehow, the yogstation coder "ktlwjec" managed.

Overall, this PR gets a 6/10 from me

File: code/game/machinery/suit_storage_unit.dm (hunk 1 above)
Comment (alexkar598) [reply code/game/machinery/suit_storage_unit.dm:60]: Modularisation is no longer enforced.

**Review by alexkar598 — APPROVED** [review]

**Comment by HomocidlePancakes** [comment]
Port prebase RnD when?

**Pull request merged**